### PR TITLE
Removing get

### PR DESCRIPTION
**Pull request by spekulatius**
I refactored the code to use the SilverStripe request instead of the PHP $_GET.

### CODE REVIEW

**Comment by spekulatius** [comment]
Hello @tractorcow 

do you want to merge this in?

Spek

File: code/model/MultiForm.php [hunk 1]
```diff
@@ -205,9 +205,12 @@ public function getCurrentStep() {
 
 		// Determine whether we use the current step, or create one if it doesn't exist
 		$currentStep = null;
-		if(isset($_GET['StepID'])) {
-			$stepID = (int)$_GET['StepID'];
-			$currentStep = DataObject::get_one('MultiFormStep', "\"SessionID\" = {$this->session->ID} AND \"ID\" = {$stepID}");
+		$StepID = Controller::curr()->request->getVar('StepID');
```
Comment (tractorcow) [inline code/model/MultiForm.php]: Should we not use $this->controller?

File: code/model/MultiForm.php [hunk 2]
```diff
@@ -205,9 +205,12 @@ public function getCurrentStep() {
...
+		if(isset($StepID)) {
+			$currentStep = DataObject::get_one(
+				'MultiFormStep',
+				"\"SessionID\" = {$this->session->ID} AND \"ID\" = {$StepID}"
```
Comment (tractorcow) [inline code/model/MultiForm.php]: Please use the ORM methods (`->filter()`) instead of unescaped sql values.

**Comment by spekulatius** [comment]
Hello @tractorcow 

I have fixed those last points.

spek

**Comment by tractorcow** [comment]
I'd love to merge this, but any chance we could get those commits squashed? 5 commits for a minor change is a bit massive. :)

**Comment by dhensby** [comment]
hmm, we have a couple of merge commits, too...

It might be worth resetting the branch and cherry-picking your changes on top of the latest code?

**Comment by tractorcow** [comment]
Done!

**Pull request merged**